layouts/partials/head.html

```diff
@@ -52,4 +52,46 @@
     navigator.serviceWorker.register('/sw.js', { scope: '/' })
   }
   </script>
+  {{ if .IsPage }}
+  {{ if eq .Type "blog" }}
+  <script type="application/ld+json">
+  {
+    "@context" : "http://schema.org",
+    "@type" : "BlogPosting",
+    "articleSection" : "{{ .Section }}",
+    "name" : "{{ .Title | safeJS }}",
+    "headline" : "{{ .Title | safeJS }}",
+    "description" : "{{ if .Description }}{{ .Description | safeJS }}{{ else }}{{if .IsPage}}{{ .Summary  }}{{ end }}{{ end }}",
+    "inLanguage" : "en-US",
+    "author" : "{{ .Params.author }}",
+    "creator" : "{{ .Params.author }}",
+    "accountablePerson" : "{{ .Params.author }}",
+    "copyrightHolder" : "{{ .Params.author }}",
+    "copyrightYear" : "{{ .Date.Format "2006" }}",
+    "datePublished" : "{{ .Params.date }}",
+    "publisher": "Sauce Labs",
+    "logo": "https://opensource.saucelabs.com/images/appicons/icon-128x128.png",
+    "url" : "{{ .Permalink }}",
+    "wordCount" : "{{ .WordCount }}",
+    {{ if .Params.image }}"image" : "{{ .Params.image | absURL }}",{{ end }}
+    "keywords" : [ "{{ range $i, $e := .Params.tags }}{{ if $i }}", "{{ end }}{{ $e }}{{ end }}" ]
+  }
+  </script>
+
+  <!-- Open Graph Metadata -->
+  {{ template "_internal/opengraph.html" . }}
+
+  <meta name="twitter:domain" value="opensource.saucelabs.com" />
```

Comment:
There is a Twitter [template](https://gohugo.io/templates/internal/#twitter-cards) as well, I thought I had mentioned it before. Not sure if all the attributes are there but I see `twitter:card` is [there](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/twitter_cards.html#L2), same as above, we should not duplicate this logic.